src/ontology/metadata/upheno.sssom.config.yml

```diff
@@ -0,0 +1,40 @@
+curie_map:
+  MP: http://purl.obolibrary.org/obo/MP_
+  HP: http://purl.obolibrary.org/obo/HP_
+  WBPhenotype: http://purl.obolibrary.org/obo/WBPhenotype_
+  XPO: http://purl.obolibrary.org/obo/XPO_
+  PLANP: http://purl.obolibrary.org/obo/PLANP_
+  ZP: http://purl.obolibrary.org/obo/ZP_
+  DPO: http://purl.obolibrary.org/obo/FBcv_
+  FYPO: http://purl.obolibrary.org/obo/FYPO_
+  DDPHENO: http://purl.obolibrary.org/obo/DDPHENO_
+  phipo: http://purl.obolibrary.org/obo/PHIPO_
+  MGPO: http://purl.obolibrary.org/obo/MGPO_
+  APO: http://purl.obolibrary.org/obo/APO_
+subject_prefixes:
+  - MP
+  - HP
+  - WBPhenotype
+  - XPO
+  - PLANP
+  - ZP
+  - DPO
+  - FYPO
+  - DDPHENO
+  - phipo
+  - MGPO
+  - APO
+
+relations:
+  - oboInOwl:hasDbXref
```

Comment:
```suggestion
```

Reply:
I dont know what this does but we do not want matches based on hasDbXref in this dataset